shell/platform/android/platform_view_android_jni.cc

```diff
@@ -46,9 +46,11 @@ bool CheckException(JNIEnv* env) {
 
 static fml::jni::ScopedJavaGlobalRef<jclass>* g_flutter_callback_info_class =
     nullptr;
-static fml::jni::ScopedJavaGlobalRef<jclass>* g_flutter_view_class = nullptr;
-static fml::jni::ScopedJavaGlobalRef<jclass>* g_flutter_native_view_class =
-    nullptr;
+
+// FlutterJNI.java, used in 2nd iteration of embedding to centralize all JNI
```

Comment:
I'd drop the part about the "2nd iteration", when someone reads the code there's just this one jni interface.

Reply:
I removed the comment